athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/propertygraph/rowwriters/CustomSchemaRowWriter.java

```diff
@@ -105,20 +113,26 @@ public static void writeRowTemplate(RowWriterBuilder rowWriterBuilder, Field fie
                             } 
                             else {
                                 Object fieldValue = obj.get(fieldName);
+                                logger.debug("writeRowTemplate VARCHAR*" + field.getName() + "*" + minorType + "*"
+                                        + (fieldValue == null ? "" : fieldValue.getClass()) + "*");
 
                                 if (fieldValue != null) {
                                     if (fieldValue.getClass().equals(String.class)) {
                                         value.value = fieldValue.toString();
                                         value.isSet = 1;
                                     }
-                                    else {       
+                                    else if (fieldValue instanceof ArrayList) {
```

Comment:
Do we need to apply this condition `else if (fieldValue instanceof ArrayList)` to rest of types?  For example `DATEMILLI`